aten/src/ATen/native/quantized/cpu/qnnpack_maxpool.cpp

```diff
@@ -0,0 +1,157 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <aten/src/ATen/native/Pool.h>
```

Comment:
do we need the "aten/src" part? seems a bit weird to not have it for the other two ATen headers but have it for this one...